commands/manifest/template_test.go

```diff
@@ -211,7 +426,91 @@ func Test_template(t *testing.T) {
 				t.Fatal(err)
 			}
 			g.Assert(strings.Contains(string(templated), "mountPath: /azure-bucket")).IsTrue("the spec should contain volumeMounts")
-			fmt.Println(string(templated))
+			// fmt.Println(string(templated))
+		})
+		g.It("Should template a manifest file with Chart and raw yaml", func() {
+			g.Timeout(100 * time.Second)
+			manifestFile, err := ioutil.TempFile("", "gimlet-cli-test")
+			if err != nil {
+				t.Fatal(err)
+			}
+			defer os.Remove(manifestFile.Name())
+			templatedFile, err := ioutil.TempFile("", "gimlet-cli-test")
+			if err != nil {
+				t.Fatal(err)
+			}
+			defer os.Remove(templatedFile.Name())
+
+			ioutil.WriteFile(manifestFile.Name(), []byte(manifestWithChartAndRawYaml), commands.File_RW_RW_R)
+			args = append(args, "-f", manifestFile.Name())
+			args = append(args, "-o", templatedFile.Name())
+
+			err = commands.Run(&Command, args)
+			g.Assert(err == nil).IsTrue(err)
+
+			templated, err := ioutil.ReadFile(templatedFile.Name())
+			g.Assert(err == nil).IsTrue(err)
+			if err != nil {
+				t.Fatal(err)
+			}
+			g.Assert(strings.Contains(string(templated), "type: LoadBalancer")).IsTrue("the service spec should contain type: LoadBalancer ")
```

Comment:
perhaps add one more assertation. One that checks the presence of the manifest, and one for the templated chart.